build.vsh

```diff
@@ -75,10 +84,9 @@ fn build(mode ReleaseMode, explicit_debug bool) {
 		println('Release mode is recommended for production use. At runtime, it is about 30-40% faster than debug mode.')
 	}
 
-	res := mode.compile()
```

Comment:
given this change, .compile() is not used anymore, and can be potentially removed.

Reply:
Right. Thanks!